about/index.html

```diff
@@ -26,7 +27,7 @@
     <div class="mzp-c-navigation">
       <div class="mzp-c-navigation-l-content">
         <div class="mzp-c-navigation-container">
-          <div class="mzp-c-navigation-logo"><a id="wordmark" href="/">Mozilla</a></div>
+          <div class="mzp-c-navigation-logo"><a id="wordmark" href="https:ios.mozilla.org">Mozilla</a></div>
```

Comment:
This link is wrong

Reply:
Doh, I forgot the //, but in general its better if this is not relative, so every user and crawler at the end gets to iot.mozilla.org and not the duplicate content on x.github.io

```suggestion
          <div class="mzp-c-navigation-logo"><a id="wordmark" href="https://iot.mozilla.org">Mozilla</a></div>
```

Reply:
That change looks good.